pilot/pkg/security/authz/model/helper.go

```diff
@@ -63,7 +63,7 @@ func fullPermission(tag string) Permission {
 
 func fullPrincipal(tag string) Principal {
 	return Principal{
-		User:          "user-" + tag,
+		Users:         []string{"user-" + tag},
```

Comment:
"user-" -> "users-".

Reply:
@pitlv2109 this one is marked as resolved but not updated in the PR?

Reply:
hmm not sure, but it should have it now.

Reply:
Yeah, just saw the new updates. thanks.